packages/doenetml/src/Viewer/renderers/angle.tsx

```diff
@@ -205,23 +207,16 @@ export default React.memo(function Angle(props) {
             board.updateRenderer();
         }
 
-        return (
-            <>
-                <a name={id} />
-            </>
-        );
+        return null;
```

Comment:
My thought was that, for objects that are in graphs, one might still want to `<ref>` them (or maybe `<xref>` then if we convert to PreTeXt's tag). In that case, we'd still want something in the DOM at that location so the browser can scroll to where the graph is. If we are removing the `<a>` anchors, should we put in a `<span id={id} />` so the browser can locate it? Same for point/button/booleanInput/math and other things that go in graph. (I don't feel too strongly about it.)

Reply:
Yes, a `span` with an id should be fine.